source/upgrading-django.rst

```diff
@@ -5,39 +5,40 @@ Django routinely releases new versions containing bug and
 security fixes, and not infrequently new major versions with
 interesting new features that we want to use.
 
-Upgrading an existing project to use a newer version of
-Django is always a little different, but there are some
-principles that are almost always helpful.
+This document outlines:
 
-One major version at a time
----------------------------
+- some principles that that we have found helpful when upgrading
+  Django
+- a list of steps that we have used when upgrading Django on
+  different projects
 
-If you're multiple major versions behind, do the Django upgrade one
-major version at a time, working through all the upgrade notes for
-one release and resolving all issues before moving on to the
-next Django major version.
 
-It's okay to jump straight to the last available minor release
-of a major version though.
+Helpful Principles
+------------------
+
+Upgrade from LTS version to LTS version
```

Comment:
I like this change. One question... the text here assumes that you are currently on an LTS version. Should we add a sentence along the lines of "If you're not on an LTS, upgrade to the nearest LTS first"?

Reply:
Sounds like a good idea